packages/babel-plugin-check-es2015-constants/README.md

```diff
@@ -56,4 +56,4 @@ require("babel-core").transform("code", {
 
 ## Note
 
-This check will only validate consts. If you need it to compile down to `var` then you must also install and enable [`transform-es2015-block-scoping`](../babel-plugin-transform-es2015-block-scoping).
```

Comment:
Yeah, we have to choose in cases like this whether it's more weird to:

- Jump to website when viewing on GitHub (or npm, etc)
- Jump to GitHub repo when viewing on website